src/copyright-headers.ts

```diff
@@ -0,0 +1,44 @@
+import { cdk, TextFile } from "projen";
+
+export class CopyrightHeaders {
+  constructor(project: cdk.JsiiProject) {
+    // Run copywrite tool to add copyright headers to all files
+    // This runs on individual repositories that use this Projen template
+    project.buildWorkflow?.addPostBuildSteps(
+      {
+        name: "Setup Copywrite tool",
+        uses: "hashicorp/setup-copywrite@3ace06ad72e6ec679ea8572457b17dbc3960b8ce", // v1.0.0
+        with: { token: "${{ secrets.GITHUB_TOKEN }}" },
+      },
+      { name: "Add headers using Copywrite tool", run: "copywrite headers" }
+    );
+
+    // copywrite config for the individual pre-built providers
+    new TextFile(project, ".copywrite.hcl", {
+      lines: `schema_version = 1
+
+project {
+  license        = "MPL-2.0"
+  copyright_year = 2021
```

Comment:
Technically this should be whatever the year the prebuilt provider repo was created, though in practice having the wrong year here doesn't really have any consequences.

Related: https://github.com/hashicorp/terraform-cdk/issues/2449

Reply:
I'm going to resolve this for now since this seems to read like a nice to have, but we can create a new issue to track this work if it needs to be done (please let me know if I should). I'd think that also finding the repo creation year dynamically might be tough to do and might not be worth the effort.